### PR TITLE
Камелина - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by JuliaKamelina**
@AnastasiaaSenina @UsovaMA please review

### CODE REVIEW

File: code/statistical_values/model/statistics.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class Statistics:
+    def __init__(self, values=None):
+        if isinstance(values, (int, float)):
+            self.values = [values]
+        elif isinstance(values, (list, tuple)):
+            self.values = values
+        else:
+            raise TypeError
+
```
Comment (AnastasiaaSenina) [inline code/statistical_values/model/statistics.py:9]: Please add error description.

File: code/statistical_values/model/statistics.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class Statistics:
+    def __init__(self, values=None):
```
Comment (AnastasiaaSenina) [inline code/statistical_values/model/statistics.py:2]: Please make the class name  and method more descriptive.

File: code/statistical_values/model/statistics.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        m = len(tmp)//2
+        if len(tmp) % 2 == 0:
+            res = (tmp[m] + tmp[m-1])/2
+        else:
+            res = tmp[m]
+        return res
+
+    def bmoment(self, k):
+        if k <= 0:
+            raise ValueError
```
Comment (AnastasiaaSenina) [inline code/statistical_values/model/statistics.py:33]: Some

**Review by AnastasiaaSenina — CHANGES_REQUESTED** [review]

File: code/statistical_values/model/test_statistics.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+    def test_raises_on_empty(self):
+        with self.assertRaises(TypeError):
+            Statistics()
+
+
+class TestStatisticsOperations(unittest.TestCase):
+    def test_can_compute_mean(self):
+        stat = Statistics([1, 2, 3, 2])
+        self.assertEqual(stat.mean(), 2.0)
```
Comment (UsovaMA) [inline code/statistical_values/model/test_statistics.py:35]: Split tests into blocks:
Arrange
Act
Assert

**Comment by UsovaMA** [comment]
Actually I suggested separating Act from Assert...

**Comment by UsovaMA** [comment]
:+1:

**Review by AnastasiaaSenina — APPROVED** [review]
👍

**Comment by JuliaKamelina** [comment]
@kirill-korniakov

**Review by sawirani — APPROVED** [review]

**Pull request merged**